sugar/certificates.go

```diff
@@ -0,0 +1,50 @@
+package sugar
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/ydb-platform/ydb-go-sdk/v3/internal/xerrors"
+)
+
+func LoadCertificatesFromFile(caFile string) ([]*x509.Certificate, error) {
+	if len(caFile) > 0 && caFile[0] == '~' {
```

Comment:
In common libraries doesn't need shell expand.
I suggent to use caFile as path "as is"

Reply:
ack